openapi-annotation-processor/src/test/kotlin/io/javalin/openapi/processor/UserCasesTest.kt

```diff
@@ -97,5 +97,19 @@ internal class UserCasesTest : OpenApiAnnotationProcessorSpecification() {
             .isEqualTo("getApiPandaList")
     }
 
+    @OpenApi(
+        path = "/api/panda/{pandaId}/name/<startsWith>",
```

Comment:
I think it might be a good idea to support parameters with `-` separator like `panda-id`/`starts-with`. I think it's even more common way to name parameters.

Reply:
I'm not a fan of hyphens in parameter names, since most syntaxes do not allow for them to exist there. Nevertheless, what would you expect? `/api/panda/{pandaId}/name/<startsWith>`'s auto-generated `operationId` would result in `apiPandaByPandaIdNameByStartsWith` ?

Reply:
Yeah, the output should be pretty much the same for both. In Javalin it's quite common, even official docs refers such usage, e.g. here:

* https://javalin.io/documentation#crudhandler

so that's definitely something we should cover.

Reply:
Oh, I wasn't aware of it. Yes, I agree we should cover the hyphens.